server/src/services/getBiodataSearch.service.ts

```
@@ -0,0 +1,117 @@
+import { CustomReq } from '../interfaces/express';
+import { PersonalAllDetailsModel } from '../models/PersonalAllDetails.Model';
+
+export const getBiodataSearch = async (req: CustomReq) => {
+  // Get page and limit from query params
+  const page = parseInt(req.query._page as string) || 1;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)